MinimalMVVM/ViewModel/DelegateCommand.cs

```diff
@@ -12,18 +12,10 @@ public DelegateCommand(Action action)
             _action = action;
         }
 
-        public void Execute(object parameter)
-        {
-            _action();
-        }
+        public void Execute(object parameter) => _action();
```

Comment:
I'm not a big fan of using expression syntax for methods. To me it seem less clear than the traditional syntax.